router/router.go

```diff
@@ -0,0 +1,18 @@
+package router
+
+import (
+	"blog/handlers"
```

Comment:
Here also need to add comments to the packages like third party or build-in or user defined